e2e/test/scenarios/models/model-actions.cy.spec.js

```diff
@@ -381,6 +299,162 @@ describe(
   },
 );
 
+['postgres', 'mysql'].forEach((dialect) => {
+  describe(`Write actions on model detail page (${dialect})`, () => {
+
+    beforeEach(() => {
+      cy.intercept("GET", "/api/card/*").as("getModel");
+
+      resetTestTable({ type: dialect, table: WRITABLE_TEST_TABLE });
+      restore(`${dialect}-writable`);
+      cy.signInAsAdmin();
```

Comment:
Aren't we already signed in as admin at this point? I think top-level `beforeEach` does that

Reply:
no. this describe() is not nested inside anything.